uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -222,6 +226,21 @@ define([
       };
 
       /**
+       * Removes attachment. Also, sets the flag to let UI know that a removal has happened.
+       * @param {String} fromWhere source from where deletion is being made like
+       * from files array or fileuploader's queue
+       * @param {Object} file
+       */
+      this.removeAttachment = function (fromWhere, file) {
+        if (fromWhere === 'files') {
+          this.request.deleteAttachment(file);
+        } else if (fromWhere === 'queue') {
```

Comment:
correct and what we discussed in 1-on-1 chat was that since there is no default there cant be a empty else condition